contrib/debian/rules

```diff
@@ -1,24 +0,0 @@
-#!/usr/bin/make -f
-# -*- mode: makefile; coding: utf-8 -*-
-
-#DEB_MAKE_CHECK_TARGET = test_bitcoin
-#build/bitcoind::
-#	$(if $(filter nocheck,$(DEB_BUILD_OPTIONS)),,src/test_bitcoin)
-
-DEB_INSTALL_EXAMPLES_bitcoind += debian/examples/*
-DEB_INSTALL_MANPAGES_bitcoind += debian/manpages/*
-
-%:
-	dh --with bash-completion $@
-
-override_dh_auto_clean:
-	if [ -f Makefile ]; then $(MAKE) distclean; fi
-	rm -rf Makefile.in aclocal.m4 configure src/Makefile.in src/bitcoin-config.h.in src/build-aux src/qt/Makefile.in src/qt/test/Makefile.in src/test/Makefile.in
```

Comment:
As I said, we're using a slightly different build process than Bitcoin is, so we don't need [this file](https://raw.githubusercontent.com/bitcoin/bitcoin/master/contrib/debian/rules). 

The script `build-debian-package.sh` assumes the binaries have already been built via `build.,sh`.